CONTRIBUTING.md

```diff
@@ -80,6 +84,11 @@ mypy --ignore-missing-imports --scripts-are-modules --pretty .
 
 ## Testing
 
+FairScale code is tested on Python 3.9.7, CUDA 11.2 and the following three PyTorch versions:
```

Comment:
Please add the current versions we are supporting here or in the README. We should have it written down somewhere.

Reply:
added versions to the readme and also added a note in the release.md to check the versions are correct before each release.